### PR TITLE
test-infra/experiment/kind-conformance-e2e.sh => kind/hack/ci/e2e.sh

### PR DESCRIPTION
**Pull request by hh**
Looks like we've progressed beyond the kind conformance experiment.

### CODE REVIEW

**Comment by neolit123** [comment]
@hh try running ./hack/verify-config.sh locally.

**Comment by hh** [comment]
Ran and inspecting.

```
./hack/verify-config.sh
INFO: Analysed target //config/jobs/kubernetes-security:genjobs (1 packages loaded, 783 targets configured).
INFO: Found 1 target...
Target //config/jobs/kubernetes-security:genjobs up-to-date:
  bazel-bin/config/jobs/kubernetes-security/linux_amd64_stripped/genjobs
INFO: Elapsed time: 3.753s, Critical Path: 0.25s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
INFO: Running command line: bazel-bin/config/jobs/kubernetes-security/linux_amd64_stripped/genjobs '--config=/home/hippie/go/src/k8s.io/test-infra/INFO: Build completed successfully, 1 total action
3131c3131
<         - cd ./../../k8s.io/kubernetes && source ./../../sigs.k8s.io/kind/hack/e2e.sh
---
>         - cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh

FAILED: config is not correct, please run 'hack/update-config.sh'
```

File: config/jobs/kubernetes/sig-testing/conformance-e2e.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ presubmits:
             - "bash"
             - "--"
             - "-c"
-            - "cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh"
+            - "cd ./../../k8s.io/kubernetes && source ./../sigs.k8s.io/kind/hack/e2e.sh"
```
Comment (hh) [inline config/jobs/kubernetes/sig-testing/conformance-e2e.yaml]: This is the line that 'hack/update-config.sh' is failing on.

```
INFO: Running command line: bazel-bin/config/jobs/kubernetes-security/linux_amd64_stripped/genjobs '--config=/home/hippie/go/src/k8s.io/test-infra/prow/config.yaml' '--jobs=/home/hippie/go/src/k8s.io/test-infra/config/jINFO: Build completed successfully, 1 total action
3131,3132c3131
<         - cd ./../../k8s.io/kubernetes && source ./../../sigs.k8s.io/kind/hack/e2e.sh
<           && echo ? #added temporary echo to ensure it was this line
---
>         - cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh

FAILED: config is not correct, please run 'hack/update-config.sh'
```

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/conformance-e2e.yaml]: kubernetes-security is auto generated by hack/update-config.sh

File: config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml [hunk 2]
```diff
@@ -200,59 +200,4 @@ presubmits:
         hostPath:
           path: /sys/fs/cgroup
           type: Directory
-  # conformance test against kubernetes release-1.11 branch with `kind`, skipping
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml]: why is this removed?

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml]: note this is a sigs.k8s.io/kind presubmit, not a kubernetes/kubernetes presubmit

Comment (hh) [reply config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml]: I thought 1.11 wasn't supported any more. I've left it in in the next commit/push.

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml]: it is for kubernetes proper, but kind is still tentatively supporting it :sweat_smile: 
I don't think we enforce sig subprojects to abandon 1.11?

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 3]
```diff
@@ -3125,7 +3125,7 @@ presubmits:
         - bash
         - --
         - -c
-        - cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh
+        - cd ./../../k8s.io/kubernetes && source ./../../sigs.k8s.io/kind/hack/e2e.sh
```
Comment (BenTheElder) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: Oops, this is missing ci/

**Comment by BenTheElder** [comment]
actually in fact, the previous script is for using the conformance image, which the replacement script does NOT do, and this is only used by those jobs. the rest of our CI jobs already use the replacement script otherwise.
